CHANGELOG.asciidoc

```diff
@@ -38,6 +38,7 @@ This release also includes changes from <<release-3-6-8, 3.6.8>>.
 * Added getter method to `ConcatStep`, `ConjoinStep`, `SplitGlobalStep` and `SplitLocalStep` for their private fields.
 * Gremlin Server docker containers shutdown gracefully when receiving a SIGTERM.
 * TINKERPOP-3080 Support to specify Operator as a reducer in withSideEffect when parsing with the grammar
+* TINKERPOP-3110 Fixed bug in Bytecode build logic for strategies
```

Comment:
the JIRA reference should not be in the CHANGELOG (i know its in the entry above, but that's not correct either). JIRA references are added on release in a separate listing.